tests/props/host/host.c

```diff
@@ -165,29 +170,46 @@ int main(int argc, const char* argv[])
         oe_put_err("oe_create_enclave(): result=%u", result);
 
     /* Check expected enclave property values */
-    if (is_signed)
+    switch (test_type)
     {
+    case 0: /* unsigned */
+        _check_properties(
+            &properties,
+            !!test_type,
+            1234,                                        /* product_id */
+            5678,                                        /* security_version */
+            OE_SGX_FLAGS_DEBUG | OE_SGX_FLAGS_MODE64BIT, /* attributes */
+            512,                                         /* num_heap_pages  */
+            512,                                         /* num_stack_pages */
+            4);                                          /* num_tcs */
+        break;
+    case 1: /* signed */
         _check_properties(
             &properties,
-            is_signed,
+            !!test_type,
```

Comment:
Similarly, this works, but the intent is unclear, consider instead:
```suggestion
            (test_type != TEST_PROPS_UNSIGNED),
```

also on line 200.